.gitignore

```diff
@@ -4,7 +4,6 @@
 # Ignore some folders
 node_modules
 /static/framework
-dist
```

Comment:
Oh, sorry, I can push it back, but aren't all patterns supposed to have dist ignored on package-level? At least at the moment they all have it, so this change will be for all, no?